core/src/main/java/org/mobilitydata/gtfsvalidator/table/AnyTableLoader.java

```diff
@@ -21,13 +22,27 @@
 import org.mobilitydata.gtfsvalidator.parsing.CsvRow;
 import org.mobilitydata.gtfsvalidator.parsing.FieldCache;
 import org.mobilitydata.gtfsvalidator.parsing.RowParser;
+import org.mobilitydata.gtfsvalidator.validator.FileValidator;
 import org.mobilitydata.gtfsvalidator.validator.SingleEntityValidator;
 import org.mobilitydata.gtfsvalidator.validator.ValidatorProvider;
 import org.mobilitydata.gtfsvalidator.validator.ValidatorUtil;
 
 public final class AnyTableLoader {
 
   private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+  private static final List<Class<? extends FileValidator>> validatorsWithParsingErrors =
```

Comment:
You should consider renaming `validatorsWithParsingErrors` to `singleFileValidatorsWithParsingErrors`, just to be consistent with `singleEntityValidatorsWithParsingErrors`

Reply:
Renamed!